### DISCUSSION TITLE
Getting P4002 error : The schema of the introspected database was inconsistent.

### DISCUSSION THREAD

**Discussion by Devr-pro** [prisma/prisma#18589] [Q&A] [answered]
Hello there, am super new to prisma and have setup prisma with `pnpx prisma init` and had run the `pnpx prisma db pull` but got the following error :
![prisma-P4002](https://user-images.githubusercontent.com/89377107/229176151-ac7a015f-de29-4f1a-b230-a4444e699d2f.jpg)
(I have supabase as my backend, having `bookmarks,datas,enhanceforms and infos` as my tables in the database)
Thanks!

**Comment by nurul3101** [comment, marked as answer]
Hello @Devr-pro 👋 

Have you included the `auth` schema in the schemas attribute in the datasource section in schema file?
It should look something like this
```prisma
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
  schemas  = ["auth", "__OTHER_SCHEMAS__"]
}
```

**Reply by Devr-pro** [reply]
Hey, thanks for the reply
Yes I have tried to do that but my editor complains about it that it can only contain with multipleschema property, What does that mean?

**Comment by Devr-pro** [comment]
Well, it simply  worked after I added the `previewFeatures = ["multiSchema"]` property to generator client 
and `schemas = ["auth","public"]` to datasource db

**Reply by nurul3101** [reply]
I am glad to hear 🙌
